### DISCUSSION TITLE
Option for ridable to have name

### DISCUSSION THREAD

**Discussion by Netherpwn** [PurpurMC/Purpur#734] [__Dismissed__] [answered]
An option to require player to use ridable mobs to have nametags. For instance, a player can ride an rgb sheep named "_jeb" but they can't ride other normal sheep.

**Comment by BillyGalbreath** [comment]
Stuff like this is best suited for a plugin. There are events you can listen to, like the [EntityMountEvent](https://purpurmc.org/javadoc/org/spigotmc/event/entity/EntityMountEvent.html) that you can cancel if your conditions are not met.

**Comment by YouHaveTrouble** [comment, marked as answer]
Implemented in [PurpurExtras](https://github.com/YouHaveTrouble/PurpurExtras/releases/tag/v1.8) :)
Have fun!
